src/Lumi/Components2/Link.purs

```diff
@@ -56,15 +53,17 @@ link =
             , className
             , href: un URL props.href
             , onClick:
-              handler (merge { button, metaKey, altKey, ctrlKey, shiftKey, syntheticEvent }) \{ button, metaKey, altKey, ctrlKey, shiftKey, syntheticEvent } -> do
-                case props.navigate, button, metaKey, altKey, ctrlKey, shiftKey of
-                  Just n', Just 0, Just false, Just false, Just false, Just false ->
-                    runEffectFn1
-                      (handler (stopPropagation <<< preventDefault) $ const n')
-                      syntheticEvent
-                  _, _, _, _, _, _ ->
-                    runEffectFn1
-                      (handler stopPropagation mempty)
-                      syntheticEvent
+                handler (merge { button, metaKey, altKey, ctrlKey, shiftKey, syntheticEvent }) \{ button, metaKey, altKey, ctrlKey, shiftKey, syntheticEvent } -> do
+                  case props.navigate, button, metaKey, altKey, ctrlKey, shiftKey of
+                    Just n', Just 0, Just false, Just false, Just false, Just false ->
+                      runEffectFn1
+                        (handler (stopPropagation <<< preventDefault) $ const n')
+                        syntheticEvent
+                    _, _, _, _, _, _ ->
+                      runEffectFn1
+                        (handler stopPropagation mempty)
+                        syntheticEvent
             , target: toNullable props.target
+            , tabIndex: props.tabIndex
+            , download: toNullable props.download
```

Comment:
@spicydonuts don't we need a `download` in our `Button` as well?

Reply:
This is an attribute html anchor tags support. To download the file with a button it'd have to insert an invisible anchor tag into the page somewhere and "click" it. That version of the download behavior is probably better as a standalone function we can assign to `onPress`.

Reply:
It looks like this: https://github.com/lumihq/purescript-lumi-components/pull/134/commits/0b3c3e64eea97648865a60d1fb394a887e6cc103#diff-9584469154ca8d1677f0dbf3ff2749ceR19

It's probably best to use an anchor tag styled as a button in this case.